chap-cheri-x86-64.tex

```diff
@@ -373,6 +374,17 @@ \subsubsection{Absolute Addresses}
 register.  These operands are always evaluated as offsets relative to
 \DDC{} including in capability mode.
 
+\subsubsection{Direct Memory-Offset MOVs}
+
+The direct memory-offset \insnnoref{MOV} instructions store the
+absolute address of a memory operand as an immediate operand.
+Extending these instructions to support capability immediates would
+require padding nops to align the capability immediate.  However, we do
```

Comment:
as well as text relocations, even for position-dependent code, which is the bigger nope